addons/test_mail/tests/test_performance.py

```diff
@@ -172,7 +172,7 @@ def test_adv_activity_full(self):
             'default_res_model': 'mail.test.activity',
         })
 
-        with self.assertQueryCount(__system__=10, emp=16):  # com runbot: 9 - 15 // test_mail only: 10 - 15
+        with self.assertQueryCount(__system__=10, emp=17):  # com runbot: 9 - 15 // test_mail only: 10 - 15
```

Comment:
wait, what?

why is your branch modifying the query count related to mails??

Reply:
The enterprise commit changes the create of activities which affects those tests.

Reply:
ha I hadn't see the enterprise branch >_<

Reply:
I think that it may be possible to avoid this additionnal query but unfortunatelly we need to merge that today. I'll check that in details once I'm back.